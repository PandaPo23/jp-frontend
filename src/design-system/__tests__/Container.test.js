import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';

import Container from '../Container';

describe('Container component', () => {
  it('should render Container component', () => {
    const wrapper = mount(<Container />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
