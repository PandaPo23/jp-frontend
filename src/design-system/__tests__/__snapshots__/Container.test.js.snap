// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component should render Container component 1`] = `
.emotion-0 {
  margin-left: auto;
  margin-right: auto;
  max-width: 40rem;
}

@media screen and (min-width:40em) {
  .emotion-0 {
    max-width: 52rem;
  }
}

@media screen and (min-width:52em) {
  .emotion-0 {
    max-width: 64rem;
  }
}

@media screen and (min-width:64em) {
  .emotion-0 {
    max-width: 80rem;
  }
}

<Container
  maxWidth={
    Array [
      "40rem",
      "52rem",
      "64rem",
      "80rem",
      "104rem",
      "128rem",
    ]
  }
  mx="auto"
>
  <div
    className="emotion-0"
  />
</Container>
`;
